PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# Pull Request Template
+
+## Description
+
+Describe the changes that where made in this pull request. When possible start with a user story - short, simple descriptions of a feature told from the perspective of the person who desires the new capability. Be sure to also include the following information:
+
+
+1. Motivation for change
+2. What was changed
+3. Link to relevant issues and documentation
+
+Example:
+*As a Blockstack developer, I would like to encrypt files using the app private key. This is needed because storing unencrypted files is unacceptable. This pull request adds the* `*encryptContent*` *function which will take a string and encrypt it using the app private key. For details refer to issue #123* 
+
+
+## Type of Change
+- [ ] New feature
```

Comment:
Maintaining a [changelog](https://keepachangelog.com) for new features and bug fixes can be useful. Should we add one? If so, should we require PRs to update the changelog, or it could be something we batch update before releases.

Reply:
Agreed, we should add a change log as well.